.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
What is this line for?

Reply:
[YAML uses three dashes (“---”) to separate directives from document content. This also serves to signal the start of a document if no directives are present. Three dots ( “...”) indicate the end of a document without starting a new one, for use in communication channels.](https://yaml.org/spec/1.2/spec.html#id2760395)